src/components/SiteHeader/SiteHeader.jsx

```diff
@@ -16,20 +16,47 @@ const HEADER_HIDDEN = {
 /**
  * JSX element for Barnes logos.
  */
-const Logo = ({ size, width, height }) => (
-  <svg className={`a-logo__svg a-logo__svg--${size}`} width={width} height={height} aria-labelledby='logo-title'>
+const Logo = ({ size, width, height, logo, className }) => (
+  <svg className={`a-logo__svg a-logo__svg--${size} ${className ? className : ""}${size === 's' ? "H" : ""}`} width={width} height={height} aria-labelledby='logo-title'>
     <title id='logo-title'>Barnes</title>
-    <use xlinkHref={`#icon--logo-${size}`}></use>
+    <use xlinkHref={`#icon--logo-${size === 's' ? "100H" : logo ? "100" : size}`}></use>
   </svg>
 );
 
+
 // Static info mapped to static Logo JSX element.
-const LOGOS = [
+const logoSizes = [
   { size: 's', width: 121, height: 37 },
   { size: 'm', width: 146, height: 45 },
   { size: 'l', width: 164.958, height: 50 },
   { size: 'xl', width: 200, height: 62 },
-].map((logo => <Logo key={logo.size} {...logo} />));
+];
+
+// keep width constant, scale height
+const logo100Sizes = [
+  { size: 's', width: 263.66, height: 37 },
+  { size: 'm', width: 146, height: 70.32 },
+  { size: 'l', width: 164.958, height: 79.45 },
+  { size: 'xl', width: 200, height: 95.75 },
+];
```

Comment:
I would update this to include a logo key and only include the width or height keys when we want them used:
```
const logo100Sizes = [
  { size: 's', height: 37, logo: "100H" },
  { size: 'm', height: 45, logo: "100H" },
  { size: 'l', width: 164.958, logo: "100" },
  { size: 'xl', width: 200, logo: "100" },
];
```
So on the stacked logos, we will keep the same width, but on the horizontal logos, we will keep the same height! This will keep everything in relatively the same proportion to the current logos!